scripts/st2bootstrap-el6.sh

```diff
@@ -186,6 +192,44 @@ install_st2web() {
   sudo chkconfig nginx on
 }
 
+install_st2chatops() {
+  # Install Node
+  curl -sL https://rpm.nodesource.com/setup_4.x | sudo -E bash -
+  sudo yum install -y nodejs
+
+  # Install st2chatops
+  sudo yum install -y st2chatops
+}
+
+configure_st2chatops() {
+  # Set credentials
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_USERNAME.).*/\1$USERNAME/" /opt/stackstorm/chatops/st2chatops.env
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_PASSWORD.).*/\1$PASSWORD/" /opt/stackstorm/chatops/st2chatops.env
+
+  # Setup adapter
+  if [ "$HUBOT_ADAPTER"="slack" ] && [ ! -z "$HUBOT_SLACK_TOKEN" ]
+  then
+    sudo sed -i.bak -r "s/^(export HUBOT_ADAPTER.).*/\1$HUBOT_ADAPTER/" /opt/stackstorm/chatops/st2chatops.env
+    sudo sed -i.bak -r "s/^(export HUBOT_SLACK_TOKEN.).*/\1$HUBOT_SLACK_TOKEN/" /opt/stackstorm/chatops/st2chatops.env
+
+    sudo service st2chatops restart
```

Comment:
@enykeev: This doesn't work

https://gist.github.com/lakshmi-kannan/27e97aa7168247044005